setup.cfg

```diff
@@ -12,7 +12,34 @@ tag_prefix =
 parentdir_prefix = great_expectations-
 
 [flake8]
-max-line-length = 200
-exclude = tests/*,docs/*
+exclude = .git,
+    build,
+    tests/*,
+    docs/*,
+    build/*,
+    assets/*,
+    contrib/*,
+    versioneer*,
+    examples/*,
```

Comment:
@Shinnnyshinshin Is the examples directory something that would be good to lint against?